src/lib.rs

```diff
@@ -53,7 +53,6 @@
 #![no_std]
 #![warn(missing_docs)]
 #![warn(rust_2018_idioms)]
-#![warn(rustdoc)]
```

Comment:
Why did you remove this?

Reply:
Has been restored in the latest submission. But, when building in the nightly version, a warning will be prompted.

```
warning: lint `rustdoc` has been removed: use `rustdoc::all` instead
  --> src/lib.rs:56:9
   |
56 | #![warn(rustdoc)]
   |         ^^^^^^^
   |
   = note: `#[warn(renamed_and_removed_lints)]` on by default

warning: 1 warning emitted
```

Reply:
Ah, based on that warning, let's just replace it with `rustdoc:all` as suggested :)

Reply:
@jonhoo If we need to eliminate all warnings, it’s better to remove it :) The stable build does not support `rustdoc::all`, and will prompt an error.

Reply:
We can skip here first.